.kitchen.yml

```diff
@@ -8,8 +8,8 @@ provisioner:
 
 platforms:
   - name: ubuntu-12.04
-  - name: centos-6.6
-  - name: centos-7.1
+  - name: centos-6.8
+  - name: centos-7.3
```

Comment:
@fahlmant I have updated the .kitchen.yml to look a bit more like other repos, in 021e7b08d5fab13
When you get time, please review and let me know.